gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/metadata/CSVMetaDataParser.java

```diff
@@ -27,28 +27,17 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.Set;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
 /**
  * Responsible for creating a {@link MetaData} instance from its string representation.
  */
-public class MetaDataParser {
-  /**
-   * Used to separate property meta data.
-   */
-  private static final String PROPERTY_DELIMITER = ",";
-  /**
-   * Used to separate property tokens (property-key, property-type)
-   */
-  private static final String PROPERTY_TOKEN_DELIMITER = ":";
+public class CSVMetaDataParser {
```

Comment:
MetaDataParser in `org.gradoop.flink.io.impl.deprecated.logicalgraphcsv` should be marked as deprecated.

Reply:
Done!